files/netdata-apcupsd.conf

```diff
@@ -0,0 +1,47 @@
+# you can disable an alarm notification by setting the 'to' line to: silent
+
+ template: apcupsd_10min_ups_load
+       on: apcupsd.load
+    class: Utilization
+     type: Power Supply
+component: UPS
+       os: *
+    hosts: *
+   lookup: average -10m unaligned of percentage
+    units: %
+    every: 1m
+     warn: $this > (($status >= $WARNING)  ? (70) : (80))
+    delay: down 10m multiplier 1.5 max 1h
+     info: average UPS load over the last 10 minutes
+       to: sitemgr
+
+# Discussion in https://github.com/netdata/netdata/pull/3928:
+# Fire the alarm as soon as it's going on battery (99% charge) and clear only when full.
```

Comment:
This appears to be too sensitive, as the battery self-test triggers this alarm.

Perhaps we can use state ONBATT instead?